File: Quality Code/Homeworks/Homework-08-High-Quality-Classes/Cohesion-and-Coupling/Geometry/Geometry2D/Geometry2DUtils.cs
namespace CohesionAndCoupling.Geometry.Geometry2D
{
    using System;

    public static class Geometry2DUtils
    {
        public static double CalcDistance2D(double x1, double y1, double x2, double y2)
        {
            double deltaX = x1 - x2;
            double deltaY = y1 - y2;

            double distance = Math.Sqrt((deltaX * deltaX) + (deltaY * deltaY));

            return distance;
        }
    }
}

File: OOP/Homeworks/Homework-09-OOP-in-Java/01.Geometry/src/geometry/geometry2d/Geometry2DUtils.java
package geometry.geometry2d;

public final class Geometry2DUtils {
    private Geometry2DUtils(){

    }

    public static double calcDistanceBetweenVertices(Vertex2D v1, Vertex2D v2) {
        double deltaX = v1.getX() - v2.getX();
        double deltaY = v1.getY() - v2.getY();

        double distance = Math.sqrt(deltaX * deltaX + deltaY * deltaY);

        return distance;
    }
}
